recipe/meta.yaml

```diff
@@ -53,13 +53,25 @@ outputs:
       host:
         - sysroot_{{ ctng_target_platform }} {{ conda_glibc_ver }}
         - {{ pin_subpackage("libgomp", exact=True) }}
+        - {{ pin_subpackage("libstdcxx-ng", exact=True) }}
+        - {{ pin_subpackage("libgcc-ng", exact=True) }}
       run:
         - {{ pin_subpackage("binutils_impl_" ~ ctng_target_platform, max_pin=None) }}
         # libstdcxx-ng is a runtime dep of gcc because LTO requires it.
         - {{ pin_subpackage("libstdcxx-ng", max_pin=None) }}
         - {{ pin_subpackage("libgcc-ng", max_pin=None) }}
         - {{ pin_subpackage("libgomp", max_pin=None) }}
         - sysroot_{{ ctng_target_platform }}
+      run_constrained:
+        - libgcc-ng !=7.3,!=9.2     # [x86_64]
+        - libstdcxx-ng !=7.3,!=9.2  # [x86_64]
+        - libgomp !=7.3,!=9.2       # [x86_64]
+        - libgcc-ng !=8.2           # [ppc64le]
+        - libstdcxx-ng !=8.2        # [ppc64le]
+        - libgomp !=8.2             # [ppc64le]
+        - libgcc-ng !=7.3           # [aarch64]
+        - libstdcxx-ng !=7.3        # [aarch64]
+        - libgomp !=7.3             # [aarch64]
```

Comment:
```suggestion
        - libgcc-ng !=5.4,!=7.2,!=7.3,!=8.2,!=9.2
        - libstdcxx-ng !=5.4,!=7.2,!=7.3,!=8.2,!=9.2
        - libgomp !=5.4,!=7.2,!=7.3,!=8.2,!=9.2
```
```

Reply:
I forgot 5.4.0 but why not specify this list per platform? 

Doing it this way means we could never build 8.2 on linux-64 even if we wanted to, for example. We only need to mask the builds we have now per platform.

Reply:
We have to make sure we conflict with defaults too. We can build 8.4.0 on linux-64 if we wanted to. No reason to go back to 8.2.

Reply:
Ah right. Defaults. Thanks!